olca-app/src/org/openlca/app/editors/graphical/model/ExchangeNode.java

```diff
@@ -9,6 +9,10 @@
 import org.openlca.core.model.ProcessLink;
 import org.openlca.core.model.descriptors.ProcessDescriptor;
 
+/**
+ * A {@link ExchangeNode} represents a flow. It could be an input or an output
```

Comment:
It represents an exchange, not a flow. An exchange is an input or output of a flow.